src/components/oscal-utils/OSCALProfileResolver.js

```diff
@@ -39,15 +39,8 @@ export default function OSCALResolveProfileOrCatalogUrlControls(
   onError,
   pendingProcesses // tracks state of recursive calls
 ) {
-  let itemUrl = origItemUrl;
+  const itemUrl = new URL(origItemUrl, parentUrl).toString();
```

Comment:
See above comment on `OSCALLinkUtils.js` -- this probably needs a check for a `TypeError` when `parentUrl` is not an absolute link.